data/mods/Magiclysm/items/integrated.json

```diff
@@ -43,5 +43,157 @@
       }
     ],
     "melee_damage": { "bash": 6 }
+  },
+  {
+    "id": "integrated_scaled_hands",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "talons" },
+    "description": "Your scales have grown to cover your hands and elongate the tips in to jagged claws that can be serviceable as cutting and piercing weapons in unarmed combat.",
+    "weight": "100 g",
+    "volume": "350 ml",
+    "qualities": [ [ "CUT", 2 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 6 ] ],
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "black_dragon_scales" ],
+    "symbol": ";",
+    "color": "dark_gray",
+    "flags": [ "INTEGRATED", "ALLOWS_NATURAL_ATTACKS", "UNBREAKABLE", "OUTER", "PADDED", "NO_SALVAGE" ],
+    "armor": [
+      {
+        "material": [ { "type": "black_dragon_scales", "covered_by_mat": 100, "thickness": 3 } ],
+        "covers": [ "hand_l", "hand_r" ],
+        "coverage": 100,
+        "encumbrance": 5
+      }
+    ],
+    "melee_damage": { "cut": 5, "stab": 5 }
+  },
+  {
+    "id": "integrated_demon_claws",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "talons" },
+    "description": "Your index fingers have become talons.  They make for wicked natural daggers.  Unfortunately, they get in the way of everything including gloves.",
+    "weight": "100 g",
+    "volume": "350 ml",
+    "qualities": [ [ "CUT", 2 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 12 ] ],
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "demon_chitin" ],
+    "symbol": ";",
+    "color": "dark_gray",
+    "flags": [ "INTEGRATED", "ALLOWS_NATURAL_ATTACKS", "UNBREAKABLE", "OUTER", "PADDED", "NO_SALVAGE" ],
+    "armor": [
+      {
+        "material": [ { "type": "demon_chitin", "covered_by_mat": 100, "thickness": 3 } ],
+        "covers": [ "hand_l", "hand_r" ],
+        "specifically_covers": [ "hand_fingers_l", "hand_fingers_r" ],
+        "coverage": 10,
+        "encumbrance": 3
+      }
+    ],
+    "melee_damage": { "cut": 10 }
+  },
+  {
+    "id": "integrated_dragon_talons_black",
```

Comment:
what happens when you have both talons and the scale armor?  Do you're hands gain duplicate encumbrance and armor?

Reply:
I think I can make hand don't include scale at the fingers, only have hide at that part so it have room for talons?